build.sh

```diff
@@ -229,7 +226,7 @@ release_ci() {
     git fetch pcengines -t
     git checkout -f $1
     git submodule update --init --checkout
-    check_if_legacy $(git describe --tags --abbrev=0 ${1})
+    check_version $(git describe --tags --abbrev=0 ${1})
```

Comment:
```suggestion
    check_version $(git describe --tags --abbrev=0 ${1})
    check_if_legacy $(git describe --tags --abbrev=0 ${1})
```
we still need to check for legacy, otherwise `legacy=$?` will not work